tests/foreman/ui/test_contentview.py

```diff
@@ -1642,6 +1642,7 @@ def test_positive_remove_cv_version_from_default_env(session, module_org):
         assert ENVIRONMENT not in cvv['Environments']
 
 
+@skip_if_bug_open('bugzilla', 1729153)
```

Comment:
I think above Bug is not applicable for this test , can you please recheck ? also I have seen that this test did not fail in last snap too.

Reply:
Hmm let correct that